packages/css/src/heading/README.md

```diff
@@ -0,0 +1,20 @@
+# Heading
+
+Een gebruiker zou middels headings een goede indruk moeten krijgen van de content van een webpagina. Een heading zegt iets over de onderstaande content.
+
+## Toegankelijkheid
+
+Headings en de heading structuur zijn vooral belangrijk voor screen readers en zoekmachines.
+
+## Richtlijnen
+
+- Één `h1` tag per pagina die het hoofdonderwerp van de pagina beschrijft.
```

Comment:
Do we want to make this a rule? I don't think it's necessary for a11y, semantics, or SEO. At [onderzoek.amsterdam.nl](https://onderzoek.amsterdam.nl/) we use 3 `h1`s, corresponding to the `header`, `main` and `footer` sections of the page.

Reply:
So many opinions about this on the internet :) Let's leave it out. There are no WCAG rules against it.